Api/ResourceResolver.php

```diff
@@ -21,7 +21,7 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Samuel ROZE <samuel.roze@gmail.com>
  */
-trait ResourceResolverTrait
```

Comment:
Traits are evil because it's an hidden copy/pasting of code ! Moreover, when a trait depends on an attribute that **needs** to be populated by the trait "class user" it increase complexity and ends with an implicit dependency. The inverse is quite true, looks that the [CollectionNormalizer](https://github.com/dunglas/DunglasApiBundle/pull/146/files#diff-612309671a94bd9a33fe3d2ab1e899fcL50) that was dependent to the `ResourceCollection` object but never using it... except though this little evil `use` statement :stuck_out_tongue: